eng/tools/npm-admin-tasks.ps1

```diff
@@ -0,0 +1,60 @@
+param (
```

Comment:
I'd suggest putting this in the scripts directory for consistency with all the other languages. However it looks like JS already has most of the scripts in the tools directory. I'm not sure it makes that big of a difference so we can keep it here.